docs/user_guide.md

```diff
@@ -556,76 +531,184 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
+
+   ![Project scanner tab](img/select-scanner.png)
+1. Select the **Repositories** tab and select a repository.
+
+   For each tag in the repository, the **Vulnerabilities** column displays the vulnerability scanning status and related information.
+   
+   ![Tag vulnerability status](img/tag-vulnerability-status.png)
+1. Select a tag, or use the check box at the top to select all tags in the repository, and click the **Scan** button to run the vulnerability scan on this image.
+
+   ![Scan an image](img/scan_image.png)
+
+   **NOTE**: You can start a scan at any time, unless the status is **Queued** or **Scanning**. If the database has not been fully populated, there is a warning message at the footer of the **Projects** > **Repositories** view and you cannot run the scan. The following statuses are displayed in the **Vulnerabilities** column:
+   
+   * **Not Scanned:** The tag has never been scanned.
+   * **Queued:** The scanning task is scheduled but has not run yet.
+   * **Scanning:** The scanning task is in progress and a progress bar is displayed.
+   * **View log:** The scanning task failed to complete. Click **View Log** link to view the related logs.
+   * **Complete:** The scanning task completed successfully.
+
+   If the process completes successfully, the result indicates the number of vulnerabilities with each severity level and the number of vulnerabilities that are fixable.
+
+   ![Scan result](img/scan-result.png)
+
+   * **Red:** Critical level of vulnerabilities
+   * **Orange:** High level of vulnerabilities
+   * **Yellow:** Medium level of vulnerabilities
+   * **Blue:** Low level of vulnerabilities
+   * **Green:** No vulnerability
+   * **Grey:** Unknown level of vulnerabilities
+1. Hover over the number of fixable vulnerabilities to see a summary of the vulnerability report. 
+
+   ![Vulnerability summary](img/vulnerability-summary.png)
+1. Click on the tag name to see a detailed vulnerability report.
+ 
+  ![Vulnerability report](img/tag_detail.png)
+  
+   In addition to information about the tag, all of the vulnerabilities found in the last scan are listed. You can order or filter the list by the different columns.
+
+### Scan All Images
+
+In addition to scanning individual images in projects, you can run global scans on all of the images in a Harbor instance, across all projects.
+
+1. Log in to the Harbor interface with an account that has Harbor administrator privileges.
+1. Expand **Administration**, and select **Interrogation Services**. 
+1. Select the **Vulnerability** tab and click **Scan Now** to scan all of the images in all projects.
+
+   ![Scan all images](img/scan_all.png)
+   
+Scanning requires intensive resource consumption. To avoid frequently triggering scans too frequently, scans can be only triggered once in a defined period. If scanning is unavailable, the next available time is displayed next to the **Scan Now** button.
```

Comment:
>To avoid frequently triggering scans too frequently, scans can be only triggered once in a defined period. 

Not available anymore. It the last `scan all` process is ongoing, the button should be disabled and the API will reject the `scan all` request if the last one is not completed.

Reply:
@steven-zou reworded to "Scanning requires intensive resource consumption. If scanning is in progress, the **Scan Now** button is unavailable" in https://github.com/goharbor/harbor/pull/10029/commits/a1eac99a49aa192cb5048c5eaa8a4e2a756cd134.